### ISSUE TITLE
VK Music refresh token

### ISSUE THREAD

**Issue by oklookat** [oklookat/synchro#9]
Auth pacakge maded, but idk how to refresh tokens. Without refresh after ~80 days user must make reauth. Thats not good.

**Comment by oklookat** [comment]
related: https://github.com/oklookat/vkmauth

**Comment by oklookat** [comment]
I didn't find anything about this in the mobile app queries. Perhaps somewhere deep in the code the token lifetime is checked and if it expires, a request is made to the server. Should I wait 80 days for the token to expire? Or reverse the app itself looking for clues (spoiler: I didn't find any).

There are mentions of the Refresh method in the VK SDK for Android. That's it. Just a mention. The method is empty.

You can find repositories in GitHub that mention getting a GMS ID, some registrations, then sending it all off to somewhere like /method/tokens.Refresh, and so on. It's a lot of work. Plus, does this have anything to do with VK Music? Those repositories are more for BOOM, which is kind of an older version of VK Music.

In short, the situation is similar to #5. So we'll probably count on VK ID reverse and reauthorisation after the token expires.
